package.json

```diff
@@ -4,7 +4,7 @@
   "description": "login.gov static site",
   "scripts": {
     "build-js": "webpack --config webpack.config.js --progress --color",
-    "federalist": "npm run build-js",
+    "pages": "npm run build-js",
```

Comment:
Slack confirmation of renamed script: https://gsa-tts.slack.com/archives/C1NUUGTT5/p1664896538701049

Related code: https://github.com/cloud-gov/pages-build-container/blob/staging/src/steps/build.py#L186-L200